_episodes/01-numpy.md

```diff
@@ -96,19 +149,18 @@ weight in kilograms is now: 65.0
 ~~~
 {: .output}
 
-If we imagine the variable as a sticky note with a name written on it,
-assignment is like putting the sticky note on a particular value:
+A variable is analoguous to a sticky note with a name written on it:
+assigning value to a variable is like putting that sticky note on a particular value.
```

Comment:
Typo: analoguous
Should be "assigning a value"

Reply:
thank you